sample_code/ep544/rev05/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM bitnami/git
+RUN --mount=type=ssh GIT_SSH_COMMAND="ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no" git clone git@github.com:asottile/pyupgrade
```

Comment:
Using additional GIT_SSH_COMMAND setup, because otherwise git clone fails. This is the noninteractive mode for git for what I know.